src/components/atomic/InputLabel/__tests__/InputLabel.tests.js

```diff
@@ -18,11 +18,10 @@ describe('', function () {
 		/>
 	);
 
-	test('InputLabel renders with correct class names and label hint', function () {
-		const { getByTestId, getByText } = wrapper;
-		const label = getByTestId(`tid-${htmlFor}-label`);
+	it('InputLabel renders with correct class names and label hint', function () {
```

Comment:
Input Label is redundant here.  I always read the test description as a continuation of the describe text, is that what we should be doing?

Reply:
I wrote this one. Agreed, it's redundant here. Would be nice we follow "it('should...')" type language.